src/pages/login/login.js

```diff
@@ -118,6 +122,17 @@ const FormikSign = ({ classes, setAuthentication, props }) => (
 
         setAuthentication({ ...response.login });
 
+        const client = createAuthenticatedClient({ token: response.login.token });
+
+        client
```

Comment:
It could be cool to create a new try / ctach for the client request, because if the user requests fail for some reason, the authentication it's not locked